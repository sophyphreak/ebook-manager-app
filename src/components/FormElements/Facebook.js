import React, { Component } from 'react';
import TextAreaInput from './MainElements/TextAreaInput';

const Facebook = ({
  label,
  facebook,
  onFacebookChange,
  ...props
}) => (
  <TextAreaInput  
    label={label}
    value={facebook}
    onChange={onFacebookChange}
    {...props}
  />
);

export default Facebook;