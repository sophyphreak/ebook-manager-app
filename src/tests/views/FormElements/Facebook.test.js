import React from 'react';
import { shallow } from 'enzyme';
import Facebook from '../../../views/FormElements/Facebook';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Facebook />);
});

test('should render Facebook correctly', () => {
  expect(wrapper).toMatchSnapshot();
});
