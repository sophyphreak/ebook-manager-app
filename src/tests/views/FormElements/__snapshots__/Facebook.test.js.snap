// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Facebook correctly 1`] = `<TextAreaInput />`;
